docs/en/observability/logs-parse-filter.asciidoc

```diff
@@ -0,0 +1,1227 @@
+      {
+        "_index": ".ds-logs-example-default-2023.08.16-000001",
+        "_id": "bE4oAIoBl7fe5ItIixuC",
+        "_score": 1,
+        "_source": {
+          "host": {
+            "ip": "192.168.1.104"
+          },
+          "@timestamp": "2023-08-08T13:45:15.004Z",
+          "message": "Debugging connection issue.",
+          "log": {
+            "level": "DEBUG"
+          }
+        }
+      },
+      {
+        "_index": ".ds-logs-example-default-2023.08.16-000001",
+        "_id": "bU4oAIoBl7fe5ItIixuC",
+        "_score": 1,
+        "_source": {
+          "host": {
+            "ip": "192.168.1.102"
+          },
+          "@timestamp": "2023-08-08T13:45:16.005Z",
+          "message": "User changed profile picture.",
+          "log": {
+            "level": "INFO"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-stream-range-query]]
+==== Range queries
+
+Use {ref}/query-dsl-range-query.html[range queries] to query logs in a specific range. 
+
+The following command searches for IP addresses greater than or equal to `192.168.1.100` and less than or equal to `192.168.1.102`.
+
+[source,console]
+----
+GET logs-example-default/_search
+{
+  "query": {
+    "range": {
+      "host.ip": {
+        "gte": "192.168.1.100",<1>
+        "lte": "192.168.1.102"<2>
+      }
+    }
+  }
+}
+----
+<1> Greater than or equal to `192.168.1.100`.
+<2> Less than or equal to `192.168.1.102`.
+
+You'll get the following results only showing logs in the range you've set:
+
+[source,JSON]
+----
+{
+  ...
+  },
+  "hits": {
+    ...
+      {
+        "_index": ".ds-logs-example-default-2023.08.16-000001",
+        "_id": "ak4oAIoBl7fe5ItIixuB",
+        "_score": 1,
+        "_source": {
+          "host": {
+            "ip": "192.168.1.101"
+          },
+          "@timestamp": "2023-08-08T13:45:12.123Z",
+          "message": "Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          }
+        }
+      },
+      {
+        "_index": ".ds-logs-example-default-2023.08.16-000001",
+        "_id": "bU4oAIoBl7fe5ItIixuC",
+        "_score": 1,
+        "_source": {
+          "host": {
+            "ip": "192.168.1.102"
+          },
+          "@timestamp": "2023-08-08T13:45:16.005Z",
+          "message": "User changed profile picture.",
+          "log": {
+            "level": "INFO"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-stream-ip-ignore-malformed]]
+=== Ignore malformed IP addresses
+
+When you're ingesting a large batch of log data, a single malformed IP address can cause the entire batch to fail. Prevent this by setting `ignore_malformed` to `true` for the `host.ip` field. Update the `host.ip` field to ignore malformed IPs using the {ref}/indices-put-mapping.html[update mapping API]:
+
+[source,console]
+----
+PUT /logs-example-default/_mapping
+{
+  "properties": {
+    "host.ip": {
+      "type": "ip",
+      "ignore_malformed": true
```

Comment:
As the user is using the ECS templates, I would assume `ignore_malformed` is already set. ++ on introducing `ignore_malformed` but maybe we can just explain it with an example. @felixbarny Can you confirm it is set by default?

Reply:
I updated the component templates definition in the index template section to better explain this flag and removed the standalone section.